containers/cuda/Containerfile

```diff
@@ -1,10 +1,9 @@
 # SPDX-License-Identifier: Apache-2.0
 
-FROM nvcr.io/nvidia/cuda:12.3.2-devel-ubi9
-RUN dnf install -y python3.11 git python3-pip make automake gcc gcc-c++
+FROM nvcr.io/nvidia/cuda:12.4.1-devel-ubi9
+RUN dnf install -y python3.11 python3.11-devel git python3-pip make automake gcc gcc-c++
```

Comment:
Why do we need `python3.11-devel` all of the sudden?

Reply:
@leseb  Good question, I should have mentioned it! aarch64 was wanting Python.h to build. So adding the headers with `python3.11-devel` fixes the dependency. Like a 300k package. ubi9 defaults to 3.9 so had to make it explicit to match paytohn3.11.

Reply:
Thanks for the clarification!